.travis.yml

```diff
@@ -0,0 +1,51 @@
+language: scala
+jdk: oraclejdk8
+
+env:
+  global:
+    - SBT_GHPAGES_COMMIT_MESSAGE='Publishing Scaladoc [ci skip]'
+    - LibreSSL_VERSION=2.2.7
+    - LibreSSL_HOME=$HOME/.bin/libressl/${LibreSSL_VERSION}
+
+
+script:
+  - SBT_OPTS="-XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled -XX:MetaspaceSize=256M -XX:MaxMetaspaceSize=512M -Xmx2G"
+  - sbt checkFormat clean coverage +test
+  - |
+    if [ "$TRAVIS_BRANCH" == "master" ]; then
+      libressl_bin=${LibreSSL_HOME}/bin/openssl
+      # install LibreSSL
+      if [ ! -f "${libressl_bin}" ]; then
+        wget https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-${LibreSSL_VERSION}.tar.gz -P $HOME
+        tar -xvf $HOME/libressl-${LibreSSL_VERSION}.tar.gz -C $HOME
+        cd $HOME/libressl-${LibreSSL_VERSION}
+        ./configure --prefix=${LibreSSL_HOME} --with-openssldir=${LibreSSL_HOME} && make -j3 && make install
+        cd $OLDPWD
+        rm $HOME/libressl-${LibreSSL_VERSION}.tar.gz && rm -rf $HOME/libressl-${LibreSSL_VERSION}
+      fi
+      ${LibreSSL_HOME}/bin/openssl version
+      echo -e "Host github.com\n\tStrictHostKeyChecking no\nIdentityFile ~/.ssh/travisci-deploy-key\n" >> ~/.ssh/config
+      ${LibreSSL_HOME}/bin/openssl aes-256-cbc -K ${fill.your.key} -iv ${fill.your.iv} -in .travis/travisci-deploy-key.enc -out .travis/travisci-deploy-key -d
```

Comment:
Fail at this line:
> /home/travis/.travis/job_stages: line 70: ${fill.your.key}: bad substitution

https://travis-ci.com/pathikrit/better-files/builds/73775221#L4467